update-minor-versions.mjs

```diff
@@ -0,0 +1,35 @@
+import { execSync } from "child_process";
+import fs from "fs";
+
+function readJson(filepath) {
+  return JSON.parse(fs.readFileSync(filepath, "utf-8"));
+}
+
+function bumpStudioPlugin() {
+  console.log("... bumping studio-plugin");
+  execSync("npm version minor -w=packages/studio-plugin");
+  const packageJson = readJson("./packages/studio-plugin/package.json");
+  const newVersion = packageJson?.version;
+  if (!newVersion) {
+    throw new Error("Could not parse studio-plugin version from package.json");
+  }
+  return newVersion;
+}
+
+function bumpStudio(newVersion) {
+  console.log(`... bumping studio to ${newVersion}`);
+  const packageJsonPath = "./packages/studio/package.json";
+  const packageJson = readJson(packageJsonPath);
+  // `npm i @yext/studio-plugin@${newVersion} --save-exact` does not update the package json,
+  // likely because the new version does not exist yet.
+  packageJson.dependencies["@yext/studio-plugin"] = newVersion;
+  fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2));
+  execSync(`npm version ${newVersion} -w=packages/studio`);
```

Comment:
should this be `npm version minor -w=packages/studio` like it is for studio-plugin in case the two packages are on different minor versions?

Reply:
I was thinking we'd want them to enforce always being in sync but either works!